acceptance-tests.md

```diff
@@ -54,10 +54,13 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
             2. They can navigate to an allocation, set the status to “Denied”, and update the allocation
         2. Disabling an allocation will delete the associated OpenShift namespace, which differs from OpenStack behavior which simply disables the project.
 3. **Manage a project as a PI.**
-    * _As a PI, I should be able to manage and share my project with others on the team, but no one except the PI and the administrator should be able to remove the project._
-    * <span style="text-decoration:underline;">Steps</span>
+    * _As a PI, I should be able to manage and share my project with others on the team, but no one except the the administrator should be able to remove the project._
+    * Steps
         1. A PI can add keycloak users to a ColdFront project under the ‘users’ section in the given project ([https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/#adding-and-removing-user-from-the-project](https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/#adding-and-removing-user-from-the-project))
-        2. From here, a PI can set the user to a particular role. It seems the ‘manager’ role is the one that lets users create and remove allocations by delegating PI role/responsibilities. Some other roles should be set for other general users who are only going to use the allocated NERC resources.
+        2. From here, a PI can set the user to a particular role. It seems the ‘manager’ role is the one that lets users create and remove allocations by delegating PI role/responsibilities.
```

Comment:
It sounds like there are two roles - 'manager' and 'user'. Is it worth defining what a 'user' can do?

Reply:
Thanks for the suggestion @tzumainn , I have defined what a user and manager can do.